free-1.md

```diff
@@ -57,12 +104,31 @@ making language design decisions), as opposed to "systems" aspects of the
 project (e.g., implementing a complicated semantics that doesn't require a lot
 of language design)?_
 
+It seems to me that the split would probably be 50-50.
+It may be that the language is actually very straightforward, and just difficult to implement.
+However, I believe that would not be the case. The language needs to be O(n) effort, meaning that
+adding the info for a new law or ruling should not increase in work as the overall size of the
+database increases. It also needs to be both easy enough to use, not mired in a programmer's
+perspective, and still powerful enough to be useful.
+I think the language would require a lot of work.
+
+
 
 ### Scope
 _How big an idea is this? How ambitious is this project?_
 
+The issue with this project is that finding out 
+"This just basically not theoretically possible if you want it to be satisfyingly powerful at all"
+is a reasonable possibility. This project could basically turn out to either be manageable
+or way, way too ambitious. It's hard to say.
+
 
 ### Benefits and drawbacks
 _Why might this be a good idea for a project? Why might this not be a good idea 
 project?_
 
+It's a good idea because it would be useful, and that is the primary reason.
+It is a bad idea because it requires looking at legal documents!
+Additionally, there's a high chance that the only truly feasible manifestation of the
+language would just be trivial.
+
```

Comment:
I think this is a great idea and can definitely work as a project. The scope is large, so it would take a little bit (but not much) work to narrow it down. For example, you could start by finding some laws / codes that are written in 2-5 pages of English text (perhaps traffic codes?), and see if you could create a language that covers them. Alternatively, you could look at things such as the Harvey Mudd student catalog or sports rules, which are smaller than an entire set of laws. Alternatively, alternatively, you could pare down what the semantics do, as you've already done. For example, perhaps when a program runs, we get only a network of related cases (if the corpus of legal work is cases). An idea that combines two pared-down domains: look at a specific, small set of laws/policies/codes _over time_. If you can capture the essence of these in logic, then can you also say which changes actually changed the meanings of the laws?

There is at least one other piece of related work: [The Representation Of An Evolving System Of Legal Concepts](http://ijcai.org/Past%20Proceedings/IJCAI-81-VOL%201/PDF/050.pdf)

Also, Kevin was thinking about a similar idea.